src/main/java/sg/ncl/MainController.java

```diff
@@ -1825,7 +1824,6 @@ public String experiments(Model model, HttpSession session) throws WebServiceRun
             }
         }
 
-//        model.addAttribute("qn", )
         model.addAttribute("experimentList", experimentList);
         model.addAttribute("realizationMap", realizationMap);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "realizationMap" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)